### PR TITLE
Minor corrections in transgui.de

### PR DESCRIPTION
**Pull request by megaocta**
Fixed some rough translations

### CODE REVIEW

File: lang/transgui.de [hunk 1]
```diff
@@ -101,7 +101,7 @@ Maximum upload speed=Maximale Uploadgeschwindigkeit
 MB=MB
 Minimize to tray=Minimieren zum Tray
 Name=Name
-No host name specified=keinen Hostnamen angegeben
+No host name specified=Keinen Hostnamen angegeben
```
Comment (ajs124) [inline lang/transgui.de]: Why not "Kein Hostname angegeben". In which context does this get displayed?

Comment (leonsoft-kras) [reply lang/transgui.de]: function TConnOptionsForm.Validate: boolean;
begin
  Result:=False;
  edHost.Text:=Trim(edHost.Text);
  if Trim(edHost.Text) = '' then begin
    Page.ActivePage:=tabConnection;
    edHost.SetFocus;
    MessageDlg(**sNoHost**, mtError, [mbOK], 0);
    exit;
  end;
  edProxy.Text:=Trim(edProxy.Text);
  if tabProxy.TabVisible and cbUseProxy.Checked and (edProxy.Text = '') then begin
    Page.ActivePage:=tabProxy;
    edProxy.SetFocus;
    MessageDlg(sNoProxy, mtError, [mbOK], 0);
    exit;
  end;
  Result:=True;
end;

Comment (PeterDaveHello) [reply lang/transgui.de:104]: @leonsoft-kras I thought you replied by mistake here? Seems unrelated 😅

**Review by PeterDaveHello — APPROVED** [review]
Checked by GPT-4

**Pull request merged**